src/Nav.php

```diff
@@ -201,6 +243,23 @@ public function items(Dropdown|NavLink ...$value): self
         return $new;
     }
 
+    /**
+     * Sets the HTML attributes for the content panes of the nav component.
+     *
+     * @param array $values Attribute values indexed by attribute names.
+     *
+     * @return self A new instance with the specified pane attributes.
+     *
+     * @see {\Yiisoft\Html\Html::renderTagAttributes()} for details on how attributes are being rendered.
+     */
+    public function paneAttributes(array $values): self
```

Comment:
```suggestion
    public function paneAttributes(array $attributes): self
```

Reply:
When it is a single argument, use `value` or `values`.